vhost-device-vsock/src/txbuf.rs

```diff
@@ -55,7 +55,11 @@ impl LocalTxBuf {
         // Check if there is more data to be wrapped around
         if len < data_buf.len() {
             let remain_txbuf = &mut self.buf[..(data_buf.len() - len)];
-            data_buf.copy_to(remain_txbuf);
+            // SAFETY ok to unwrap because len < data_buf.len()
+            data_buf
+                .get_slice(len, data_buf.len() - len)
+                .unwrap()
```

Comment:
Please use `.expect()` and put the content of the comment there.

Reply:
Done